en/docs/how-tos/security/api-authentication/secure-apis-using-oauth2.0-access-tokens/secure-apis-using-jwt-self-contained-jwt.md

```diff
@@ -65,9 +66,9 @@ For information on the subscription model and configuration steps, please refer
 
 ### Passing a token to the backend.
 
- When you are using the JWT authentication, you can provide a token to be passed to the backend using a JWT claim "backendJWT". i.e. If your JWT token has **"backendJwt" claim** , then Its value is passed to the backend. To configure which header it is used for passing the backend JWT, the following should be added to the Microgateway runtime configuration in &lt;MICRO-GW-RUNTIME\_HOME&gt;/conf/micro-gw.conf.
+ When you are using the JWT authentication, you can provide a token to be passed to the backend using a JWT claim "backendJWT". i.e. If your JWT token has **"backendJwt" claim** , then Its value is passed to the backend. To configure which header it is used for passing the backend JWT, the following should be added to the Microgateway runtime configuration in `<MICRO-GW-RUNTIME_HOME>/conf/micro-gw.conf`.
```

Comment:
Please correct this. Microgateway can also generate the back end jwt. So generated jwt also passed in the below-mentioned header.